src/aind_metadata_service/tars/mapping.py

```diff
@@ -48,6 +49,14 @@ class VirusAliasPatterns(Enum):
     AIV = re.compile(r"^AiV[a-zA-Z0-9_-]+$")
 
 
+class ViralPrepAliases(BaseModel):
```

Comment:
Pydantic might be more than necessary for a small container. Probably a dataclass or a TypedDict would have sufficed.